src/components/header.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import './header.css';
 
-type pageTitles = 'Main Page' | 'About Us' | 'Not Found';
+type pageTitles = 'Main Page' | 'Forms Page' | 'About Us' | 'Not Found';
```

Comment:
Each time when you need to change page title you should also add it to this type, i think it is a bit too much, type string is enough 😃